core/crypto/bip39/impl/bip39_provider_impl.cpp

```diff
@@ -30,13 +32,46 @@ namespace kagome::crypto {
 
   Bip39ProviderImpl::Bip39ProviderImpl(
       std::shared_ptr<Pbkdf2Provider> pbkdf2_provider,
+      std::shared_ptr<CSPRNG> csprng,
       std::shared_ptr<Hasher> hasher)
       : pbkdf2_provider_{std::move(pbkdf2_provider)},
+        csprng_{std::move(csprng)},
```

Comment:
ensure is not nullptr in constructor body

Reply:
That assert broke test, where `nullptr` was used.
Replacing `nullptr` with `<mock/libp2p/crypto/random_generator_mock.hpp>`